app/views/shared/application/_phase_banner.html.erb

```diff
@@ -6,10 +6,16 @@
     <div class="govuk-phase-banner__text phase">
       This is a new service. <%= link_to 'Give feedback', feedback_path %>
     </div>
-    <%- if @current_user.present? -%>
-    <div class="govuk-phase-banner__text auth">
-      <%= current_user_info_and_logout_link current_user %>
-    </div>
-    <%- end -%>
+    <% if @current_user.present? %>
```

Comment:
One for another time, but having both `@current_user` and `@current_candidate` is a bit confusing